src/Files.App/GlobalUsings.cs

```diff
@@ -51,7 +51,7 @@
 global using global::Files.App.ViewModels;
 global using global::Files.App.ViewModels.UserControls;
 global using global::Files.App.Views;
-global using global::Files.App.Views.LayoutModes;
+global using global::Files.App.Views.ContentLayouts;
```

Comment:
There is a chance that we'll actually add a layout with this name and this would be confusing.

Reply:
How about "Layouts" then?

Reply:
![image](https://github.com/files-community/Files/assets/62196528/2001d9b2-07b2-42cb-8cb5-49ae4e9d0890)

Reply:
`Layouts` is fine